src/app/zones/views/ZoneDetailView.vue

```diff
@@ -102,14 +106,85 @@
             v-if="props.data.zoneInsight.subscriptions.length > 0"
           >
             <h2>{{ t('zone-cps.detail.subscriptions') }}</h2>
-
-            <KCard class="mt-4">
-              <SubscriptionList
-                :subscriptions="props.data.zoneInsight.subscriptions"
+            <AppCollection
+              :headers="[
+                { ...me.get('headers.zoneInstanceId'), label: 'Zone Leader Instance ID', key: 'zoneInstanceId' },
```

Comment:
Btw should these labels be i18n?

Reply:
Glad you asked, I keep meaning to find the best place to mention this.

For the moment I've kinda given up doing i18n texts unless its useful for machine > human.

Our locale files are a bit of a mess as there is no consistency, lots of repetition and no real thought gone into them. I'm kinda avoiding adding to that until we have a change to put a bit more thought into it.

It will be easier once to add these to the i18n files once we are following a pattern to avoid repetition, whereas as the locale files get bigger and bigger it will be harder and harder to start following some sort of pattern.

I think I should probably put something in an issue actually, wdyt?

Reply:
Lets keep i18ning things, structure isn't ideal at the minute but we can sort that out separately

Reply:
I made a couple of issues off the back of this:

- https://github.com/kumahq/kuma-gui/issues/2813
- https://github.com/kumahq/kuma-gui/issues/2814